conanfile.py

```diff
@@ -5,7 +5,7 @@
 
 class HomestoreConan(ConanFile):
     name = "homestore"
-    version = "4.5.10"
+    version = "4.5.11"
```

Comment:
This should be a MINOR bump (4.6.1) since we adding an new interface call. That way the consumer knows not to try to use something prior to `4.6.x`

Reply:
done